pkg/network/ebpf/c/tracer/tracer.h

```diff
@@ -43,8 +43,6 @@ typedef struct {
     __u64 sent_packets;
     __u64 recv_packets;
     __u8 direction;
-    // keep the conn_tags u8 to keep the struct slim
-    __u8 conn_tags;
```

Comment:
Why are you removing the tags?

Reply:
we don't use it anymore as Pedro merged the `protocol_stack` and forgot to cleanup this